test/visual-diff/visual-tests

```diff
@@ -600,5 +600,14 @@
       ],
       "interactive_tests": "examples/visual-tests/amp-story/amp-story-page-attachment-desktop.js"
     },
+    {
+      "url": "examples/visual-tests/amp-user-notification/amp-user-notification-blocking.amp.html",
```

Comment:
I think that it might be worth changing the file name to `amp-user-notification.amp.html`, since this is testing both blocking and unblocking, and you might want to check other things as well in later tests. Your choice of course :)